apps/anoma_lib/lib/anoma/rm/transparent/proving_systems/rlps.ex

```diff
@@ -58,7 +58,7 @@ defmodule Anoma.RM.Transparent.ProvingSystem.RLPS.Instance do
   def to_noun(instance) do
     [
       instance.tag,
-      Noun.Nounable.Bool.to_noun(instance.flag),
+      Noun.Nounable.Atom.to_noun(instance.flag),
```

Comment:
If you are using a proper protocol it is probably best to just use Nounable itself without extra qualifiers.

```suggestion
      Noun.Nounable.to_noun(instance.flag),
```